ProcessMaker/Http/Middleware/LaravelTokenMiddleware.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ProcessMaker\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use Laravel\Passport\Passport;
+
+class LaravelTokenMiddleware
+{
+    /**
+     * Handle an incoming request.
```

Comment:
In this comment, can you explain the idea and goal of the middleware?

Reply:
Sure